cartridges/int_adyen_overlay/cartridge/scripts/util/AdyenHelper.ds

```diff
@@ -664,7 +666,27 @@ var __AdyenHelper: Object = {
             }
         }
         return applicationInfo;
+    },
+
+    validateStateData: function (stateData) {
+        var validFields = ["paymentMethod", "billingAddress", " deliveryAddress", "riskData", "shopperName",
+            "dateOfBirth", "telephoneNumber", "shopperEmail", "countryCode", "socialSecurityNumber", "browserInfo",
+            "installments", "storePaymentMethod", "conversionId"]
+        var invalidFields = []
+        var filteredStateData = {}
+        var stateDataKeys = Object.keys(stateData)
+        for (var i = 0; i < stateDataKeys.length; i++) {
+            var keyName = stateDataKeys[i]
+            var includesInvalidField = validFields.indexOf(keyName) === -1
+            if (includesInvalidField) {
+                invalidFields.push(keyName)
+            } else {
+                filteredStateData[keyName] = stateData[keyName]
+            }
+        }
+        return { stateData: filteredStateData, invalidFields: invalidFields }
```

Comment:
do you use the 'invalidFields' for any purpose? Maybe it is worth to log it for debugging purposes.